src/cryptoadvance/specter/hwi_rpc.py

```diff
@@ -137,8 +137,12 @@ def _get_client(self, device_type=None, path=None, fingerprint=None, passphrase=
                     client = SpecterClient(device["path"])
                 else:
                     client = hwi_commands.get_client(device_type, path, passphrase)
+                if not client:
+                    raise Exception('The device was identified but could not be reached.  Please check it is properly connected and try again')
                 client.is_testnet = 'test' in chain
```

Comment:
'signet' is also testnet, but doesn't have 'test' in chain.
The only mainnet is 'main', so I think is_testnet should be if `chain == 'main'`

Reply:
Right, fixed